### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by sameep-s**
Added Wishlist page for e-commerce app.

Screenshot (77)

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -76,15 +79,15 @@
   color: var(--night);
 }
 
-.fa-cart-shopping,
-.fa-heart {
+.nav-cart-icon,
+.nav-heart-icon {
   font-size: 3rem !important;
```
Comment (shubhambishnoi) [inline css/navbar.css]: remove important and use specificity to apply styling

File: css/props.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: 1rem;
+  right: 1rem;
+  background-color: var(--day);
+}
+
+.card-icon-heart {
+  font-size: 2.4rem !important;
```
Comment (shubhambishnoi) [inline css/props.css:46]: remove important here also

**Review by shubhambishnoi — APPROVED** [review]

File: css/props.css (hunk 2 above)
Comment (sameep-s) [reply css/props.css:46]: yes sure i'll fix this.

File: css/navbar.css (hunk 1 above)
Comment (sameep-s) [reply css/navbar.css]: thanks for pointing it out.

Comment (sameep-s) [reply css/navbar.css]: i fixed this.

**Review by Nagashree01 — APPROVED** [review]
This is looking good to me. Avoid using !important

**Pull request merged**